tensorflow/python/kernel_tests/cast_op_test.py

```diff
@@ -19,6 +19,7 @@
 from __future__ import print_function
 
 import numpy as np
+import sys
```

Comment:
We need to import sys as we are using 'sys.byteorder' to detect endianness of underlying platform.